core/core-spacemacs.el

```diff
@@ -227,4 +227,18 @@ FILE-TO-LOAD is an explicit file to load after the installation."
        (force-mode-line-update))
      (spacemacs/check-for-new-version spacemacs-version-check-interval))))
 
+(defun spacemacs/system-info ()
+  (kill-new
```

Comment:
Documentation added in the revised PR :), please let me know if any other changes are suggested.